calico_containers/tests/unit/datastore_test.py

```diff
@@ -81,6 +84,14 @@
                  "active", "11-22-33-44-55-66")
 EP_12.profile_ids = ["UNIT"]
 
+ETCD_ENV_DICT = {
```

Comment:
Very minor point, I think you only reference this once, so perhaps just as easy to embed in the one place that uses it?